sympy/algebras/tests/test_quaternion.py

```diff
@@ -23,6 +24,22 @@ def test_quaternion_construction():
     nc = Symbol('nc', commutative=False)
     raises(ValueError, lambda: Quaternion(w, x, nc, z))
 
+@pytest.mark.parametrize(
```

Comment:
SymPy has its own test runner which is sort of a copy of pytest. While it is possible to use pytest with the sympy codebase (I always do) we do need to make sure that the tests run without pytest i.e. using the `bin/test` script when `pytest` is not installed.